asteroid/models/conv_tasnet.py

```diff
@@ -82,6 +82,12 @@ def __init__(
                 "be the same. Received "
                 f"{n_feats} and {in_chan}"
             )
+        if causal and norm_type != "cgLN":
+            norm_type = "cgLN"
+            print(
+                "In causal configuration cumulative layer normalization"
+                f"must be used. Changing f{norm_type} to cgLN"
+            )
```

Comment:
Can you use `warnings.warn()` instead? 

How about channel layer norm, is it over time as well?

Reply:
Channel Layer norm should be fine too, the mean and var is only computed on channel dim. In the paper they didn't use it though.

Reply:
I think we can just warn about these two norms and use cumLN as default in causal mode, right ?